ansible/roles/screenly/tasks/main.yml

```
@@ -11,6 +11,15 @@
     - .config
     - screenly_assets
 
+- name: Create empy .asoundrc
+  file:
```

```suggestion
  ansible.builtin.file:
```

See this [documentation](https://ansible.readthedocs.io/projects/lint/rules/fqcn/#problematic-code) about FQCN for reference. By making things more specific/explicit, there's less chance of unexpected behaviors.